secrets/portworx-with-dcos-secrets.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: page
+title: "Portworx with DC/OS Secrets"
+sidebar: home_sidebar
+meta-description: "Portworx can integrate with DC/OS Secrets to store your encryption keys/secrets. This guide will get a Portworx cluster connected to DC/OS Secrets."
+---
+
+* TOC
+{:toc}
+
+Portworx can integrate with DC/OS Secrets to store your encryption keys/secrets and credentials. This guide will get a Portworx cluster connected to DC/OS Secrets. This could be used to store secrets that will be used for encrypting Portworx volumes.
+
+>**Note:**<br/>Secrets is an DC/OS Enterprise only feature
+
+### Configuring DC/OS Secrets with Portworx
+
+During installation or when updating an existing Portworx framework, enable the feature from Secrets section.
+
+![portworx-dcos-secret](/images/dcos-portworx-secrets-setup.png){:width=2597px" height="1287px"}
+
+ The `base path` is the secrets path under which Portworx will read/write secrets. If not specified, Portworx will look for secrets at the top level.
+
+The `dcos username secret` and `dcos password secret` are the paths to secrets, where Portworx will look for credentials of the user to access the secrets. This user should have full access to secrets under the `base path`.
+
+If you want only Portworx framework to access the username and password secrets path, the path should have prefix same as Portworx service name (default service name is `portworx`).
```

Comment:
Probably put this under a sub-section called "Configuring permissions for Secrets" or something similar. It should probably be the first step since it should ideally to be done before configuring PX